esp/src/src/react/theme.ts

```diff
@@ -1,6 +1,9 @@
 import { createMuiTheme } from "@material-ui/core/styles";
 
 export const theme = createMuiTheme({
+    typography: {
+        fontSize: 15
```

Comment:
Is this large?  Would need to see where its being used...

Reply:
<img width="259" alt="pr_example_of_15" src="https://user-images.githubusercontent.com/2362956/75710058-d8785700-5c91-11ea-8501-b61efe52f831.png">